doc/README.md

```diff
@@ -0,0 +1,16 @@
+# DRIVER Documentation
+DRIVER's documentation is spread across multiple files. This lists potentially relevant files.
+
+| Document | Description |
+| - | - |
+| [**System Administration**](system-administration.md) | General purpose information needed for configuring, deploying, and administering a DRIVER environment. |
+| [**Logging**](logging.md) | List of services, which host they run on, and log file destinations. |
+| [**Troubleshooting Android app issues**](android-troubleshooting.md) | Overview of common potential problems connecting a DRIVER environment with its DRIVER Android application. |
+| [**API Documentation**](api-documentation.md) | Description of API endpoints and response structure. |
+| [**API Use Case &mdash; Health Organization**](api-use-case-health-organization.md) | Step by step example of using the DRIVER application for a hypothetical health organization. |
+| [**Batch Data Loading**](batch-data-loading.md) | Guide for loading bulk data quickly using the API or shell directly. |
+| [**Best Practices for DRIVER data**](data-best-practices.md) | Collection of best practices for defining and managing DRIVER record schemas. |
+| [**Competencies for working with a DRIVER instance**](dev_ops_competencies.md) | Definition of what tools and technologies an administrator may need to be familiar with to manage a DRIVER instance. |
+| [**Domain Management**](domain-management.md) | Guide on obtaining and configuring a domain name for your DRIVER instance. |
+| [**Stopping/Starting a Stack in AWS**](stopping-starting-aws-stack.md) | Guide for how to turning off an AWS hosted DRIVER environment when not in use. |
```

Comment:
Grammar is a little funky here. Can also mention turning it back on.

Reply:
Good catch, that is pretty mangled. How does this read?
> Guide for how to safely turn off, and later restore, an AWS hosted DRIVER environment to reduce cost when not in use.

Reply:
Perfect